gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -1320,6 +1322,81 @@
+     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#element-step" target="_blank">Reference Documentation - Element Step</a>
+     * @since 3.6.0-incubating
+     */
+    default GraphTraversal<S, Element> element() {
+        this.asAdmin().getBytecode().addStep(Symbols.element);
+        return this.asAdmin().addStep(new ElementStep<>(this.asAdmin()));
+    }
+
+    /**
+     * Perform the specified service call with no parameters.
+     *
+     * @param service the name of the service call
+     * @return the traversal with an appended {@link CallStep}.
+     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#call-step" target="_blank">Reference Documentation - Call Step</a>
+     * @since 3.6.0-incubating
+     */
+    default <E> GraphTraversal<S, E> call(final String service) {
+        this.asAdmin().getBytecode().addStep(Symbols.call, service);
+        final CallStep<S,E> call = new CallStep<>(this.asAdmin(), false, service);
+        return this.asAdmin().addStep(call);
+    }
+
+    /**
+     * Perform the specified service call with the specified static parameters.
+     *
+     * @param service the name of the service call
+     * @param params static parameter map (no nested traversals)
+     * @return the traversal with an appended {@link CallStep}.
+     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#call-step" target="_blank">Reference Documentation - Call Step</a>
+     * @since 3.6.0-incubating
+     */
+    default <E> GraphTraversal<S, E> call(final String service, final Map params) {
+        this.asAdmin().getBytecode().addStep(Symbols.call, service, params);
+        final CallStep<S,E> call = new CallStep<>(this.asAdmin(), false, service, params);
+        return this.asAdmin().addStep(call);
+    }
+
+    /**
+     * Perform the specified service call with dynanmic parameters produced by the specified child traversal.
+     *
+     * @param service the name of the service call
+     * @param childTraversal a traversal that will produce a Map of parameters for the service call when invoked.
+     * @return the traversal with an appended {@link CallStep}.
+     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#call-step" target="_blank">Reference Documentation - Call Step</a>
+     * @since 3.6.0-incubating
+     */
+    default <E> GraphTraversal<S, E> call(final String service, final Traversal<?, Map<?,?>> childTraversal) {
+        this.asAdmin().getBytecode().addStep(Symbols.call, service, childTraversal);
+        final CallStep<S,E> step = null == childTraversal ? new CallStep(this.asAdmin(), false, service) :
+                new CallStep(this.asAdmin(), false, service, new LinkedHashMap(), childTraversal.asAdmin());
+        return this.asAdmin().addStep(step);
+    }
+
+    /**
+     * Perform the specified service call with both static and dynanmic parameters produced by the specified child
```

Comment:
nit: spelling "dynamic" - perhaps also good to add a sentence to say how they merge will take place in the event of conflict, i.e. what happens when the same key is found in both maps?